src/core/js2python.c

```diff
@@ -171,18 +190,13 @@ EM_JS_NUM(errcode, js2python_init, (), {
       cache.set(obj, list);
       for (let i = 0; i < obj.length; i++) {
         entryid = Module.hiwire.new_value(obj[i]);
-        item = Module.__js2python_convert(entryid, cache, depth);
-        // clang-format off
-        if (item === 0) {
```

Comment:
If I understand correctly this moves throwing of the error into js2python_convertOther inside js2python_convert?

Reply:
That is correct.